configure.ac

```diff
@@ -194,7 +194,6 @@ AC_CHECK_HEADERS([ \
 )
 
 # Checks for typedefs, structures, and compiler characteristics.
-AC_CHECK_HEADER_STDBOOL
```

Comment:
Couldn't you use m4_ifdef here as well? Otherwise, please replace with a comment indicating AC_CHECK_HEADER_STDBOOL is in autoconf version >= 2.69. We may want to go back to 2.69+ at some point.